.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+default_language_version:
+  python: python3.9
+default_stages: [commit, push]
```

Comment:
The default is [all stages](https://pre-commit.com/#pre-commit-configyaml---top-level) which I thought runs black more times than necessary. For example, I don't think `merge` needs to run black because upstream merges would already pass the CI for black, and local merges would still run black on push to the user's fork. If you still think this should be changed, feel free to suggest deletion/commit directly to the repo.